src/domain/kaivuilmoitus/BasicInfo.tsx

```diff
@@ -30,14 +33,24 @@ export default function BasicInfo({ johtoselvitysIds }: Readonly<Props>) {
     constructionWorkChecked,
     maintenanceWorkChecked,
     emergencyWorkChecked,
+    createCableReportNotChecked,
     placementContracts,
   ] = watch([
     'applicationData.constructionWork',
     'applicationData.maintenanceWork',
     'applicationData.emergencyWork',
+    'applicationData.cableReportDone',
     'applicationData.placementContracts',
   ]);
 
+  const [showNewCableReportSection, setShowNewCableReportSection] = useState(
```

Comment:
This useState and the following useEffect are not really needed as createCableReportNotChecked could be used directly to determine if to display the radio buttons.

Reply:
Oh, great! Thank you! I guess ChatGPT is not always so reliable when trying to make things work :-)